app/viewmodels/SummaryCard.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import play.api.libs.json.Json
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+
+case class SummaryCard(
+                        cardBody: Seq[SummaryListRow],
+                        status: String,
+                        penaltyPoint: Int,
+                        footerUrl: String = ""
+                      )
+
+object SummaryCard{
+
+  implicit val fmt = Json.format[SummaryCard]
```

Comment:
This companion object is not needed as we are not reading/writing to JSON for this view model.